ansible/roles/boot-iso/tasks/dell.yml

```
@@ -2,41 +2,78 @@
 # Dell tasks for booting an iso
 
 - name: Set Virtual Media ISO
-  set_fact:
+  ansible.builtin.set_fact:
     _virtual_media_iso: "{{ virtual_media_iso | default(hostvars[item]['boot_iso']) }}"
 
+- name: "Dell - Clear iDrac job queue for {{ item }} (badfish)"
+  ansible.builtin.include_role:
+    name: badfish
+    tasks_from: call
+  vars:
+    badfish_host: "{{ hostvars[item]['bmc_address'] }}"
+    badfish_user: "{{ hostvars[item]['bmc_user'] }}"
+    badfish_password: "{{ hostvars[item]['bmc_password'] }}"
+    badfish_args:
+      - "--clear-jobs"
+      - "--force"
+    ignore_errors: true
```

I believe this ignore_errors is over indented, but even so should we permit ignoring errors on this?

---

as for "should we ignore failure": from my tests, it was hard to see actual improvements following only the job queue cleaning. The real salvation came from the iDrac reset. Cleaning the queue seems to be "nice" though, because it ensures nothing is dangling around before we start to actually fill it again. I think we may accept, for instance, an error due to iDrac being crashed at that point.

I didn't put the queue cleaning after the reset because, as said, it might not be that important (while it may help ensuring our commands are passed in a reasonable amount of time).